lib/public-api-stack/public-api-stack.js

```diff
@@ -67,6 +72,9 @@ const defaults = {
     corsPreflightMaxAgeSeconds: 600,
     logLevel: process.env.LOG_LEVEL || 'info',
     publicApiStageName: 'api',
+    hashKey: process.env.HASH_KEY || "asdf",
+    merchantId: process.env.MERCHANT_ID || "123",
+    worldlineWebhookSecret: process.env.WORLDLINE_WEBHOOK_SECRET || "secret"
```

Comment:
secrets should be imported via Secrets Manager (see AdminIdentityStack for a reference of how this is achieved).

Typically we want as little as possible being set by `process.env` as it means we have to inject the variables via GH Actions (yucky).